content/en/docs/ops/configuration/telemetry/telemetry-v1-vs-v2/index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Differences between Telemetry v1 and v2
+description: Tradeoffs and gaps between Telemetry v1 and v2.
+weight: 10
+owner: istio/wg-policies-and-telemetry-maintainers
+test: no
+---
+
+Providing uniform telemetry and extension mechanisms to extend the functionality
+in Istio have been the driving factors for the architectural choices made in
+Istio. From early days, mixer aka Telemetry v1 provided both of these
+functionalities where an out-of-process service gathered attributes from the
+workload Envoy proxies, combined it with platform data like Kubernetes services
+and deployment associations and generated rich telemetry data like Prometheus
+metrics, tracing and logs. It also provided an extension mechanism where
+users could write adapters to ingest these attributes and enforce policies or
+transform them before exporting it to a third-party monitoring tool.
+
+While this provided a flexible architecture, mixer consumed significant resources
+in clusters with large amounts of workloads and often became a single point of failure.
+Additionally, the remote call made by the mixer client at Envoy proxies
+attributed to performance degradation and telemetry loss. In Istio 1.3 and
+above, mixer-less aka telemetry v2 support was added to address these concerns
+while surfacing similar telemetry and extension capabilities as telemetry v1.
+Telemetry v2 works by adding WASM extensions in Envoy proxies for surfacing
+Prometheus metrics, thereby removing the out-of-process call to a centralized
+mixer component. Additionally, custom WASM filters can be used to provide mixer
+adapter like functionality. You can read more about this in the announcement
+[blog](/blog/2020/wasm-announce/).
+
+Moving to telemetry v2 reduces resource cost and improves proxy performance, and
+is the recommended way to gather telemetry from Istio, leading us to deprecate
+support for telemetry v1 in Istio 1.7 release. However, there are some
```

Comment:
I feel like this should be called out earlier, perhaps at the top, of this document.

Such as,

> Support for mixer telemetry in Istio 1.7 has been deprecated and will be removed in Istio 1.8. This blog details the reasons why users should adopt telemetry v2, aka mixer-less telemetry, if they haven't yet, as well as the reasons why the Istio community has decided this is the supported architecture for observability going forward.